examples/ui_priority.rs

```diff
@@ -1,5 +1,5 @@
 //! Inputs consumed by UI and not propagated to actions.
-//! In order to run this example pass `--features egui_priority,bevy_egui/render,bevy_egui/default_fonts,bevy/default_font` to cargo.
+//! In order to run this example pass `--features egui_priority,bevy_egui/render,bevy_egui/default_fonts` to cargo.
```

Comment:
FYI I think you can specify this in the example metadata. Check what the Bevy cargo.toml is doing!

Reply:
Yep, I configured it:
https://github.com/projectharmonia/bevy_enhanced_input/blob/9727118af46a6daed3ee11f97ea0e197e8d3745e/Cargo.toml#L42
Cargo just don't tell users which command to run, so I decided to clarify :)